SemiTruck.h

```diff
@@ -1,2 +1,15 @@
 #pragma once
 
+#include "Vehicle.h"
+
+struct SemiTruck : public Vehicle
+{
+    SemiTruck(const std::string& n);
+    
+    void driveSafely();
+    void pullOver();
+
+private:
+    void tryToEvade() override;
```

Comment:
Implement the special member functions as explained in `main.cpp`
remove `tryToEvade`
- [ ] revise

Reply:
Is it not ok to override tryToEvade and repurpose it for the pullOver func? Having a overridden destructor was producing some warnings